pkg/virtctl/convert-machine-type/convert_machine_type.go

```diff
@@ -0,0 +1,150 @@
+package convertmachinetype
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/utils/pointer"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+const (
+	kubevirtNamespace  = "kubevirt"
+	convertMachineType = "convert-machine-type"
+)
+
+type ConvertMachineTypeCommand struct {
+	clientConfig clientcmd.ClientConfig
+}
+
+// holding flag information
+var (
+	namespaceFlag     string
+	forceRestartFlag  bool
+	labelSelectorFlag string
+)
+
+// NewConvertMachineTypeCommand generates a new "convert-machine-types" command
+func NewConvertMachineTypeCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
```

Comment:
`virtctl convert-machine-types` is quite a top-level command. I wonder if this is too much

What about
`virtctl update convert-machine-type --namespace=bar` - for all vms in `bar` namespace
and then
`virtctl update convert-machine-type myvm ` - for a single vm

Reply:
Looks good to me!

Reply:
WDYT about: `virtctl update machine-types`? i.e. removing `convert-` from your suggestion

Reply:
Sounds good to me!